pdns/dnsdist.cc

```diff
@@ -141,6 +141,8 @@ bool g_servFailOnNoPolicy{false};
 bool g_truncateTC{false};
 bool g_fixupCase{false};
 bool g_dropEmptyQueries{false};
+uint32_t g_socketUDPSendBuffer{16777216};
```

Comment:
This is quite greedy, on OpenBSD:
```
Unable to raise socket buffer size to 16777216: No buffer space available
Unable to raise socket buffer size to 16777216: No buffer space available
```

Reply:
Right, I was afraid of something like that. I now see that the default on Linux is likely smaller than that on most distributions (212992 on x86_64, 163840 on 32 bits). So I see three options:
- a smaller default value, like the recursor does, perhaps 160k?
- default value of 0 so we keep the system default
- default value of 0 except for Linux, where we set something like 200k?

Reply:
250000 works on OpenBSD. so a default of that (or smaller) for dnsdist would work for me

Reply:
On all Linux distributions I checked, `rmem_default` == `rmem_max` and `wmem_default` == `wmem_max` by default, so we would not be able to raise the value. So I guess doing nothing by default makes more sense, and the administrator can set higher values if needed.